.github/workflows/create_discussion.yml

```
@@ -0,0 +1,126 @@
+# @usage
+#
+# on:
+#   schedule:
+#     - cron: "0 0 * * mon"
+# jobs:
+#   create_discussion:
+#     uses: route06/actions/.github/workflows/create_discussion.yml@v2
+#     with:
+#         target_day: sunday
+#         owner: route06
+#         repository_name: professional-service-div-template
+#         title: ハドル
+#         discussion_category_slug: meeting-notes
+#         description_template_path: _templates/weekly_meeting_discussion/test.md
+
+name: Create discussion
+
+on:
+  workflow_call:
+   inputs: 
+    target_day:
+      required: true
+      type: string
+    owner:
+      required: true
+      type: string
+    repository_name:
+      required: true
+      type: string
+    title:
+      required: true
+      type: string
+    discussion_category_slug:
+      required: true
+      type: string
```

提案: デフォルト値を設定してあげて、調整が不要なときは設定をしなくて済むようにしてあげた方が便利そうでした！

```suggestion
    discussion_category_slug:
      required: false
      type: string
      default: meeting-notes
```

---

以下にて対応しました！
041807d6cfef5a53c9c8e7c19e0334ecf9cc40a4